dev_guide/cron_jobs.adoc

```diff
@@ -69,25 +49,58 @@ kind: CronJob
 metadata:
   name: pi
 spec:
-  schedule: "*/1 * * * *"  <1>
-  jobTemplate:             <2>
+  schedule: "*/1 * * * *"       <1>
+  concurrencyPolicy: "Replace"  <2>
+  startingDeadlineSeconds: 200  <3>
+  suspend: false                <4>
+  successfulJobsHistoryLimit: 3 <5>
+  failedJobsHistoryLimit: 1     <6>
+  jobTemplate:                  <7>
     spec:
       template:
         metadata:
-          labels:          <3>
+          labels:               <8>
             parent: "cronjobpi"
         spec:
           containers:
           - name: pi
             image: perl
             command: ["perl",  "-Mbignum=bpi", "-wle", "print bpi(2000)"]
-          restartPolicy: OnFailure <4>
+          restartPolicy: OnFailure <9>
 ----
 
-1. Schedule for the job. In this example, the job will run every minute.
-2. Job template. This is similar to the xref:../dev_guide/jobs.adoc#creating-a-job[job example].
-3. Sets a label for jobs spawned by this cron job.
-4. The restart policy of the pod. This does not apply to the job controller. See xref:../dev_guide/jobs.adoc#creating-a-job-known-issues[Known Issues and Limitations] for details.
+<1> The schedule field for the job is specified in 
+    link:https://en.wikipedia.org/wiki/Cron[cron format]. In this example, the
+    job runs every minute.
+<2> The concurrency policy specifies how to treat concurrent jobs
+    within a cron job. Only one of the following policies can be specified. By
+    default, the policy is `Allow`.
+    * `Allow` allows multiple instances of the job to run concurrently.
+    * `Forbid`: prevents multiple instances of the job from running concurrently.
+      Scheduled runs are skipped if the previous run has not finished.
+    * `Replace`: cancels a currently running instance of the job and replaces
+    the job with a new instance.
+<3> The starting deadline specifies a deadline (in seconds) for starting the job
+    if a scheduled run is missed for any reason. After the deadline, the cron job
+    does not start the job. Missed job executions are counted as failed jobs. By
+    default, jobs have no deadline.
```

Comment:
It is important to note that if the startingDeadlineSeconds field is set (not nil), the controller counts how many missed jobs occurred from the value of startingDeadlineSeconds until now rather than from the last scheduled time until now. For example, if startingDeadlineSeconds is 200, the controller counts how many missed jobs occurred in the last 200 seconds.

Reply:
The exact implementation of counting how many missed schedules can be found : https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/

Reply:
Thanks for the feedback, I made adjustments.  A quick google search suggests that many people find this item confusing.